### PR TITLE
adev-74 - added ``` for code snippets for jukito

### PR DESCRIPTION
**Pull request by Pacane**

### CODE REVIEW

**Comment by jclariviere** [comment]
LGTM

File: site/src/main/markdown/jukito/@All.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Typical examples for this are:
 
 In order to do so all inputs must be configured in the TestModule of the test class.
 
+```
```
Comment (christiangoudreau) [inline site/src/main/markdown/jukito/@All.md]: would be nice at some point to be able to chose a language

**Comment by christiangoudreau** [comment]
LGTM

**Comment by meriouma** [comment]
LGTM. GWTP has the same problem I think.

File: site/src/main/markdown/jukito/Maven.md [hunk 2]
```diff
@@ -2,22 +2,26 @@
 
 Jukito is released on Maven central. If you use Maven you can configure add this dependency:
 
+```
 <dependency>
```
Comment (Chris-V) [inline site/src/main/markdown/jukito/Maven.md]: Does it pick up the coloring format automatically? On github you need to tell it &#096;&#096;&#096;xml &#096;&#096;&#096;

**Pull request merged**